src/Components/App/App.js

```diff
@@ -0,0 +1,107 @@
+import { Component } from 'react';
+import { getRandomFiveLetterWord, findWordInAPIDatabase } from '../../apiCalls';
+import Header from '../Header/Header';
+import GameBoardContainer from '../GameBoardContainer/GameBoardContainer';
+import './App.css';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      currentWordInPlay: null,
+      typedLetters: [],
+      submittedWords: [],
+      gameOver: false,
+      playerStats: [],
+      error: null
+    }
```

Comment:
I've not used a class component since before winter break, and I almost forgot what they look like!